include/mbedtls/cipher.h

```diff
@@ -495,9 +499,10 @@ static inline mbedtls_operation_t mbedtls_cipher_get_operation( const mbedtls_ci
  * \param operation     The operation that the key will be used for:
  *                      #MBEDTLS_ENCRYPT or #MBEDTLS_DECRYPT.
  *
- * \returns             \c 0 on success, #MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA if
- *                      parameter verification fails, or a cipher-specific
- *                      error code.
+ * \returns             \c 0 on success.
+ * \return              #MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA if
+ *                      parameter verification fails.
```

Comment:
Suggestion: Use past tense: `if [the] parameter verification failed.`.

Reply:
documentation uses present simple.

Reply:
My impression was we're more using past tense, e.g. https://github.com/ARMmbed/mbedtls/pull/1506/files/02facfb4d9a176f68bdaf8251160e52bf85a8258..b5607bf61d80d08520dd557e2041599940d2f077#diff-db0a6e576886e9c5e9e4decc01919a6cR364 or https://github.com/ARMmbed/mbedtls/pull/1506/files/02facfb4d9a176f68bdaf8251160e52bf85a8258..b5607bf61d80d08520dd557e2041599940d2f077#diff-db0a6e576886e9c5e9e4decc01919a6cR686.

Reply:
No, that just means I missed some instances. Will correct.

Reply:
Ok, thank you!